optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -169,14 +173,22 @@ public Boolean getDroolsAlphaNetworkCompilationEnabled() {
         return droolsAlphaNetworkCompilationEnabled;
     }
 
-    public KieRuntimeBuilderWrapper getGizmoKieRuntimeBuilderWrapper() {
+    public Supplier<KieBase> getGizmoKieRuntimeBuilderWrapper() {
         return gizmoKieRuntimeBuilderWrapper;
     }
 
-    public void setGizmoKieRuntimeBuilderWrapper(KieRuntimeBuilderWrapper kieRuntimeBuilderWrapper) {
+    public void setGizmoKieRuntimeBuilderWrapper(Supplier<KieBase> kieRuntimeBuilderWrapper) {
         this.gizmoKieRuntimeBuilderWrapper = kieRuntimeBuilderWrapper;
     }
 
+    public Supplier<KieBaseDescriptor> getGizmoKieBaseDescriptorWrapper() {
+        return gizmoKieBaseDescriptorWrapper;
+    }
+
+    public void setGizmoKieBaseDescriptorWrapper(Supplier<KieBaseDescriptor> gizmoKieBaseDescriptorWrapper) {
+        this.gizmoKieBaseDescriptorWrapper = gizmoKieBaseDescriptorWrapper;
+    }
```

Comment:
This is a break in public API, we can not have those unfortunately.

Reply:
The break in the public API can be reversed, since the path the code takes is different and they use different fields (one use a supplier of KieBase, another use a supplier of KieBaseDescriptor)